Chapter 8/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea folder should not be included in PR. These are your personal settings, not tasks from the book

Reply:
Done! Removed from the repository